### ISSUE TITLE
Helm chart for JobSet

### ISSUE THREAD

**Issue by avrittrohwer** [kubernetes-sigs/jobset#726]
**What would you like to be added**:

Add a helm chart for the JobSet installation.

**Why is this needed**:

Customers have expressed a preference for helm vs kubectl applying a github URL.  Example chart from Kueue: https://github.com/kubernetes-sigs/kueue/blob/1b7d3f3f71d3c49b5b02ec0766abe4c3df7cfdfd/charts/kueue/README.md

This enhancement requires the following artifacts:

- [ ] Design doc
- [ ] API change
- [x] Docs update

The artifacts should be linked in subsequent comments.

**Comment by ahg-g** [comment]
See LWS example here: https://github.com/kubernetes-sigs/lws/pull/266

**Comment by avrittrohwer** [comment]
Are there plans to publish the chart to a repo?  For example with Kueue they are pushed to us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue

**Comment by ahg-g** [comment]
Yes, see https://github.com/kubernetes-sigs/jobset/issues/790; contributions are welcome :)
